pkg/scheduler/scheduler.go

```diff
@@ -480,19 +480,19 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		// into the resources that were preempted, but this is harmless.
 		nominatedNode := ""
 		if fitError, ok := err.(*core.FitError); ok {
-			if sched.DisablePreemption {
+			if sched.DisablePreemption || !prof.HasPostFilterPlugins() {
```

Comment:
In a followup task, we should deprecate ```DisablePreemption``` since preemption can now be disabled by disabling the default preemption plugin itself.

Reply:
Yes, I listed it in the umbrella issue already.